src/interpreter.rs

```diff
@@ -2,6 +2,35 @@ use rustpython_vm::{Interpreter, Settings, VirtualMachine};
 
 pub type InitHook = Box<dyn FnOnce(&mut VirtualMachine)>;
 
+/// The convenient way to create `rustpython_vm::Interpreter` with stdlib and other stuffs.
+///
+/// Basic usage:
+/// ```
+/// let interpreter = rustpython::InterpreterConfig::new()
+///     .init_stdlib()
+///     .interpreter();
+/// ```
+///
+/// Overriding Settings:
+/// ```
+/// let settings = Settings::Default();  // override your settings here
```

Comment:
Giving an example on how to override a setting may be more beneficial to downstream users. For example…

```suggestion
/// let settings = Settings {
///     debug: true,
///     ..Settings::Default()
/// };
```